### DISCUSSION TITLE
hiding items

### DISCUSSION THREAD

**Discussion by wynteryyc** [TBog/TBLauncher#479] [Q&A]
hi

I'd like to be able to hide items that are in the dock from displaying in the apps list and I'd like hidden items to be hidden and not display the hidden icon on them

I'm probably missing something 

tia

**Comment by TBog** [comment]
I remember KISS does have the option to hide apps from favorites. I could add this feature for apps found in Dock.

1. There is an issue about hiding apps from history #455 
2. The hide feature currently hides apps from appearing in search results.

If you hide an app the way you described the only way to "un-hide" it would require adding a list of hidden apps.
Why do you want to hide apps? Maybe we can find some other solution to your problem instead of using the hide feature.

**Comment by wynteryyc** [comment]
- hide apps that come with GrapheneOS that i don't use (eg. i use a different contacts app so i'd like one to disappear)
- i think it makes sense for apps on my dock to not appear anywhere else

my 2cents

LOVE your launcher!

**Comment by TBog** [comment]
> i use a different contacts app so i'd like one to disappear

You can long click the app, choose `App info` then `DISABLE`. This is how I removed the default gallery app that came with my phone.
